README.md

```diff
@@ -79,6 +79,11 @@ Etcdadm must be run on the machine that is being added or removed. As a conseque
 On its own, etcdadm does not automate cluster operation, but a cluster orchestrator can delegate all the above tasks to etcdadm.
 
 <a name="questions"></a>
+
+## Options
+
+*  `--listen-address`, `--advertise-address`: specify an IP if you'd like something different than the auto-discovery. Regardless of this, you can also specify an alternative advertised address (for a floating IP for example).
```

Comment:
This is fine, although it conflates peer and client addresses. In the future, we should allow users to configure peer and client addresses separately.

Can you note that the address is used for derive both peer and client URLs?